README.md

```diff
@@ -84,3 +84,12 @@ const config = epinovaWebpackConfig({}, config => {
 
 module.exports = config;
 ```
+
+## Browserstack
+
+[Browserstack does not like localhost url's](https://www.browserstack.com/question/759) so it is possible to start this config with a `--browserstack` argument. Example package.json script:
+```javascript
+{
+    "develop:browserstack": "webpack-dev-server --mode development --hot --hot-only --browserstack=http://bs-local.com",
```

Comment:
Kanskje vi burde bare lagt inn et script i package.json som heter noe à la browser-test eller et annet begrep som gjør det tydelig at det er snakk om å teste løsningen i forskjellige nettlesere? "develop:browserstack" er litt for krøkkete og vi bør unngå å koble inn egennavn ;)

Reply:
I teorin ja, men är det i praktiken så att någon annan lösning kräver denna "fixen"?

Reply:
Touché. test:browserstack, kanskje? ;)